docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -43,8 +43,8 @@ The sc_create command also includes the cryptographic key to receive coins back
 The verification key guarantees that the received coins were processed according to a matching proving system. 
 Besides these parameters, sc_create has some optional ones, here is the complete set of parameters:
 
- - **version**                                    - (numeric, required) The version of the sidechain. Recommended to use version 1. For non ceasing sidechain should be 2.
- - **withdrawalEpochLength**                      - (numeric, optional, default=100) length of the withdrawal epochs. The minimum valid value in regtest is: 2, the maximum (for any network type) is: 4032. For non ceasing sidechain should be 0.
+ - **version**                                    - (numeric, required) The version of the sidechain. Recommended to use version 1. For non ceasing sidechain and for circuit with key rotation should be 2.
```

Comment:
Lets add in this file section about key rotation circuit. Describe why we need it, what it can do and which command can do it.

Also 01-scnode-api-spec.rst needs to be updated with key rotation command

Reply:
Done